app/scripts/lib/rpc-method-middleware/handlers/switch-ethereum-chain.js

```diff
@@ -80,77 +86,130 @@ async function switchEthereumChainHandler(
 
   const { origin } = req;
 
+  // setup chainId
```

Comment:
Some comments in this file merely restate what is already expressed by the code:
```suggestion
```

Reply:
done here: https://github.com/MetaMask/metamask-extension/pull/24415/commits/465ae6bbfdd4a69ef07cf122bac5fb214a4ecd98